src/extension.ts

```diff
@@ -175,16 +175,16 @@ async function activateServerForFolder(context: ExtensionContext, uri: Uri, fold
       }
     } else if (e instanceof HlsError) {
       logger.error(`General HlsError: ${e.message}`);
-      if (e.stack) {
-        logger.error(`${e.stack}`);
-      }
       window.showErrorMessage(e.message);
     } else if (e instanceof Error) {
       logger.error(`Internal Error: ${e.message}`);
+      window.showErrorMessage(e.message);
+    }
+    if (e instanceof Error) {
```

Comment:
Why do we log twice for Error?

Reply:
This is a "catch-all" for errors and always prints the stack trace of the caught exception.